blog-posts/staticIpLambda/staticIpLambda.md

```diff
@@ -0,0 +1,193 @@
+---
+published: false
+title: 'Deploying a Lambda with a static IP has never been so simple 🍰'
+cover_image:
+description: 'Learn how to deploy a lambda with a static IP (for whitelisting concerns) and perform NodeJS SFTP operations using this lambda.'
+tags: serverless, sftp, aws, lambda
+series:
+canonical_url:
+---
+
+## TL;DR <br>
+🧠 Learn how to deploy a **lambda with a static IP** (for whitelisting concerns)
+⚡ Perform **NodeJS SFTP operations** using this lambda. <br>
+
+---
+
+Your Serverless application might need to connect to a partnet server that requires IP whitelisting.
+We will use the [Serverless Framework](https://www.serverless.com/) to deploy our lambda function and the **Typescript AWS CDK** to create the infrastructure needed to give our lambda a static IP 🔒. <br>
+Moreover, we will use the NodeJS [ssh2-promise](https://www.npmjs.com/package/ssh2-promise) package to perform SFTP operations in our lambda function. 💽
+
+This is what we will build 👷:
+
+![Static Lambda IP Architecture](./assets/archi.png 'Logo Kumo')
```

Comment:
Precise that the server offers a SFTP service so that we understand why the ssh2-promise library is required.